modules/cluster-logging-rn-5.4.10.adoc

```diff
@@ -0,0 +1,31 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-10_{context}"]
+= Logging 5.4.10
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0385[OpenShift Logging Bug Fix Release 5.4.10].
+
+[id="openshift-logging-5-4-10-bug-fixes"]
+== Bug fixes
+None.
+
+[id="openshift-logging-5-4-10-CVEs"]
+== CVEs
+link:https://access.redhat.com/security/cve/CVE-2021-46848[CVE-2021-46848]
+link:https://access.redhat.com/security/cve/CVE-2022-2056[CVE-2022-2056]
+link:https://access.redhat.com/security/cve/CVE-2022-2057[CVE-2022-2057]
+link:https://access.redhat.com/security/cve/CVE-2022-2058[CVE-2022-2058]
+link:https://access.redhat.com/security/cve/CVE-2022-2519[CVE-2022-2519]
+link:https://access.redhat.com/security/cve/CVE-2022-2520[CVE-2022-2520]
+link:https://access.redhat.com/security/cve/CVE-2022-2521[CVE-2022-2521]
+link:https://access.redhat.com/security/cve/CVE-2022-2867[CVE-2022-2867]
+link:https://access.redhat.com/security/cve/CVE-2022-2868[CVE-2022-2868]
+link:https://access.redhat.com/security/cve/CVE-2022-2869[CVE-2022-2869]
+link:https://access.redhat.com/security/cve/CVE-2022-2953[CVE-2022-2953]
+link:https://access.redhat.com/security/cve/CVE-2022-2964[CVE-2022-2964]
+link:https://access.redhat.com/security/cve/CVE-2022-4139[CVE-2022-4139]
```

Comment:
@libander I don't see below security CVE's included in 5.4.10 errata link:https://access.redhat.com/security/cve/CVE-2022-2964[CVE-2022-2964]
link:https://access.redhat.com/security/cve/CVE-2022-4139[CVE-2022-4139]

Reply:
They both are. That list of CVEs was copied directly from the the errata as always.